### DISCUSSION TITLE
Multiple views, week & day views ?

### DISCUSSION THREAD

**Discussion by scramatte** [wdelfuego/nova-calendar#70] [Ideas & Feature Requests]
Hello,

When mutliple views will be available:  week view, day view ?

Regards

**Comment by bb140856** [comment]
It seams that this feature hangs on me, or to be more precise on lack of time, that can be spent on finishing this feature. Truly I cannot estimate when would be able to sit down and finish Vue views, that should be fixed. I'm open if any of community can jump into and make it done. 

I understand, that I should do this, but life is more complicated. If nobody can do this, I'll finish it ASAP, but cannot tell when "P" will come. 

Warm regards

cc: @wdelfuego

**Reply by scramatte** [reply]
Hi,

We might be interested into finish this job.  What is the implementation status?
Regards
